src/wp-includes/class-wp-theme-json-schema.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * WP_Theme_JSON_Schema class
+ *
+ * @package WordPress
+ * @subpackage Theme
+ * @since 5.9.0
+ */
+
+/**
+ * Class that migrates a given theme.json structure to the latest schema.
```

Comment:
Let's add comments in the private classes saying something like:

This class is for internal core usage and is not supposed to be used by extenders (plugins and/or themes). The global styles mechanism is a work in progress and there is a risk of breaking changes if this class is used by third-party code.

Reply:
I like that inline documentation to better explain what `@access private` means.